### DISCUSSION TITLE
requests for feature.

### DISCUSSION THREAD

**Discussion by Manvendra-div** [TomSchimansky/CustomTkinter#203] [General]
Hi Tom i am really liking your project it's Fantastic really fulfilling all needs but i really need a **filedialog** type widget i don't know if it is available ,if it's available how can i use it.

**Comment by TomSchimansky** [comment]
You can just use the tkinter file dialog, it should work fine.

**Comment by Manvendra-div** [comment]
yaa it's works very fine but it looks ugly in linux

**Comment by TomSchimansky** [comment]
To create a new file dialog is way too much work and impossible for me to do. Besides that, the file dialog should always look native on the current operating system, and the tkinter dialog should do this...

**Comment by Manvendra-div** [comment]
Yaa but i am telling linux filedailog is way too ugly anyways, i also wanna ask something if i wanted to make a file downloader with progressbar how to that i tried with requests dialog as we do with twdm but nothing seems to work.

**Comment by TomSchimansky** [comment]
You can use StackOverflow for non CustomTkinter related questions

**Comment by Manvendra-div** [comment]
It is related bro i am talking about customtkinter.CTkProgressBar
